tests/xharness/Logging/ILog.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace xharness.Logging {
+
+	// defines common log types
+	public enum LogType {
+		XML_LOG,
+		NUNIT_RESULT,
+		SYSTEM_LOG,
+		COMPANION_SYSTEM_LOG,
+		BUILD_LOG,
+		TEST_LOG,
+		EXTENSION_TEST_LOG,
+		EXECUTION_LOG,
```

Comment:
not sure why we ended up with all caps here
might be a good time to fix that if this gets re-used